src/domain/app/footer/__tests__/Footer.test.tsx

```diff
@@ -72,7 +82,69 @@ test('should show navigation links and should route to correct page after clicki
   spyWindowOpen.mockRestore();
 });
 
-test('should not show keywords and registrations link when those features are disabled', async () => {
+test('should show admin and registration links to admin user', async () => {
+  setFeatureFlags({
```

Comment:
Enabling these particular feature flags happens in four of the new test cases. Maybe you could put the tests inside a `describe` block where in a `beforeEach` you could enable the feature flags?

Reply:
Good idea. I ended up to use t.each instead to remove some more repetition from the tests